Offseason2022/src/main/java/frc/robot/commands/Climber0Stow.java

```diff
@@ -4,14 +4,31 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.Climber;
+import frc.robot.subsystems.FloorConveyor;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.TowerConveyor;
 
 /**
  *
  */
 public class Climber0Stow extends SequentialCommandGroup
 {
-  public Climber0Stow( )
+  private final Climber       m_climber;
+  private final Intake        m_intake;
+  private final FloorConveyor m_fConv;
+  private final TowerConveyor m_tConv;
+  private final Shooter       m_shooter;
+
+  public Climber0Stow(Climber climber, Intake intake, FloorConveyor fConv, TowerConveyor tConv, Shooter shooter)
   {
+    m_climber = climber;

```

Comment:
Excellent question, and it's a mistake on my part. Since the group commands only consist of a constructor, then the object doesn't need to be created as a new member variable for other methods to use. I will remove those lines in the empty commands.

Reply:
Should be fixed in latest commit. Good catch! In my haste to get all of the object parameters to be passed to the commands (to reduce how many people need to keep editing robotcontainer), I just made the group commands look like the simple commands--it was a mistake.